.github/ISSUE_TEMPLATE.md

```diff
@@ -1,3 +1,3 @@
-Please use one of the provided templates:
-- https://github.com/facebook/react-native/issues/new/choose
+## 👉 [Please follow one of these issue templates](https://github.com/facebook/react-native/issues/new/choose) 👈
```

Comment:
I've been avoiding using markdown in these templates, as they are not rendered by default and actually make it harder to copy the URL. It's fairly safe for URLs, as GitHub will still render them as links should the user hit "Preview".

For reference, this is what a user sees when opening a blank template-less issue on master right now:
![screen shot 2019-01-28 at 12 25 21 pm](https://user-images.githubusercontent.com/165856/51864233-fbe70a80-22f7-11e9-82de-2cf97688d5b0.png)

This is how the proposed change would look like:
![screen shot 2019-01-28 at 12 25 33 pm](https://user-images.githubusercontent.com/165856/51864232-fbe70a80-22f7-11e9-8af4-6cf55b9f6446.png)

This is what the current template that does not use markdown for the link looks like if the user hits "Preview":
![screen shot 2019-01-28 at 12 25 42 pm](https://user-images.githubusercontent.com/165856/51864231-fbe70a80-22f7-11e9-9440-98edd798995f.png)